sbpy/bib/core.py

```diff
@@ -29,8 +29,26 @@ def register(task, citations):
     """
```

Comment:
I think we should assume that only a single citation is provided everytime this function is called (this is the way it is setup right now) The user will not use this function manually, so we just have to call `bib.register` for each task, subtask, and reference once in our code. This will probably improve code readability. Can you change `citations` to `citation` and adjust the docstring accordingly?